packages/components/pie-tag/src/tag.scss

```diff
@@ -1 +1,152 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
```

Comment:
You shouldn't have to set box-sizing in the component styles – this gets set through `pie-css` at an application level (which is an expected dependency).